scrapers/__init__.py

```diff
@@ -1,7 +1,10 @@
 # SPDX-License-Identifier: GPL-3.0+
 # Import all scrapers here so that the scrape script can find them
-from scrapers.koji import KojiScraper  # noqa: F401
-from scrapers.distgit import DistGitScraper  # noqa: F401
-from scrapers.bugzilla import BugzillaScraper  # noqa: F401
-from scrapers.errata import ErrataScraper  # noqa: F401
-from scrapers.freshmaker import FreshmakerScraper  # noqa: F401
+from scrapers.koji import KojiScraper
+from scrapers.distgit import DistGitScraper
+from scrapers.bugzilla import BugzillaScraper
+from scrapers.errata import ErrataScraper
+from scrapers.freshmaker import FreshmakerScraper
+
+# Define the scrapers here in the order that they should be run
+all_scrapers = (BugzillaScraper, DistGitScraper, KojiScraper, ErrataScraper, FreshmakerScraper)
```

Comment:
Does order matter?

Reply:
We've written it in a way that it doesn't matter what order you run it in, but this order seems to make logical sense to me.